pkg/operator/controller.go

```diff
@@ -276,42 +289,66 @@ func (c *Controller) sync(key wqKey) error {
 	default:
 	}
 
-	// Tuned CR changed
+	// Tuned CR changed and the operator components need to be managed.
 	klog.V(2).Infof("sync(): Tuned %s", tunedv1.TunedRenderedResourceName)
 	err = c.syncTunedRendered(cr)
 	if err != nil {
 		return fmt.Errorf("failed to sync Tuned %s: %v", tunedv1.TunedRenderedResourceName, err)
 	}
 
+	if key.name != tunedv1.TunedDefaultResourceName {
+		crTuned, err := c.listers.TunedResources.Get(key.name)
+		if err != nil {
+			if !errors.IsNotFound(err) {
+				return err
+			}
+		} else if crTuned.Spec.ManagementState != "" {
+			klog.Warningf("setting ManagementState is supported only in Tuned/%s; ignoring ManagementState in Tuned/%s", tunedv1.TunedDefaultResourceName, key.name)
+		}
+	}
+
 	klog.V(2).Infof("sync(): DaemonSet")
 	err = c.syncDaemonSet(cr)
 	if err != nil {
 		return fmt.Errorf("failed to sync DaemonSet: %v", err)
 	}
 
-	if key.kind == wqKindTuned {
-		// Tuned CR changed, this can affect all profiles, list them and trigger profile updates
-		klog.V(2).Infof("sync(): Tuned %s", key.name)
+	// Tuned CR changed, this can affect all profiles, list them and trigger profile updates
```

Comment:
Why do we not need the tuned kind check anymore here?

Reply:
So I have no formal proof, but I did go through the code and also tested quite extensively (using the "else" branch -- it was never reached).  So I believe this check was probably some relic of the past code that wasn't cleaned up.  Cleaning up now.